release_bot/github.py

```diff
@@ -518,11 +526,16 @@ def put_labels_on_issue(self, number, labels):
         :param labels: list of str
         :return: True on success, False on fail
         """
+        if self.conf.dry_run:
+            self.logger.info("I would add labels to issue #" +
```

Comment:
this is the preferred way when working with loggers: logger.info("I would add labels to issue #%s", number)

Reply:
Done. I also saw that it can be done using the `{number}` notation.